packages/@react-spectrum/tag/src/Tag.tsx

```diff
@@ -10,82 +10,79 @@
  * governing permissions and limitations under the License.
  */
 
-import Alert from '@spectrum-icons/workflow/Alert';
-import {classNames, useStyleProps} from '@react-spectrum/utils';
+import {classNames, SlotProvider, useSlotProps, useStyleProps} from '@react-spectrum/utils';
 import {ClearButton} from '@react-spectrum/button';
-import {FocusRing} from '@react-aria/focus';
 import {mergeProps} from '@react-aria/utils';
-import React from 'react';
-import {SpectrumTagProps} from '@react-types/tag';
+import React, {useRef} from 'react';
 import styles from '@adobe/spectrum-css-temp/components/tags/vars.css';
+import {Text} from '@react-spectrum/text';
+import {useFocusRing} from '@react-aria/focus';
 import {useHover} from '@react-aria/interactions';
 import {useTag} from '@react-aria/tag';
-import {useTagGroupProvider} from './TagGroup';
 
-export const Tag = ((props: SpectrumTagProps) => {
+export function Tag(props) {
```

Comment:
missing its typescript, even if it's for internal use only, see
https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/menu/src/MenuItem.tsx#L35